index.bs

```diff
@@ -1628,8 +1628,8 @@ The <dfn constructor for="XRWebGLLayer">XRWebGLLayer(|session|, |context|, |laye
       <dd>
         1. Initialize |layer|'s {{XRWebGLLayer/antialias}} to |layerInit|'s {{XRWebGLLayerInit/antialias}} value.
         1. Initialize |layer|'s {{XRWebGLLayer/framebuffer}} to a new [=opaque framebuffer=] created with |context|.
-        1. Initialize a [=swap chain=] for |layer| that is compatible with |session|'s [=/XR device=].
-        1. If |layer|'s [=swap chain=] was unable to be created for any reason, throw an {{OperationError}} and abort these steps.
+        1. Allocate and initialize resources, including GPU accessible memory buffers, as required to support the compositing of |layer|.
```

Comment:
I feel it's important to keep the bit about the resources being "compatible with |session|'s [=/XR device=]", so could you add that back in? Otherwise looks good! Definitely want to expand on this more in the future but this is a cleaner way of expressing it in the meantime. Thanks!

Reply:
Fixed